compute/api/Compute.Samples/DeleteInstanceAsync.cs

```diff
@@ -23,13 +23,16 @@
 public class DeleteInstanceAsyncSample
 {
     public async Task DeleteInstanceAsync(
+        // TODO(developer): Replace these variables before running the sample.
         string projectId = "your-project-id",
         string zone = "us-central1-a",
         string machineName = "test-machine")
     {
 
-        // Initialize the client that will be used to send instance-related requests.
-        // You should reuse the same client for multiple requests.
+        // Initialize client that will be used to send requests. This client only needs to be created
+        // once, and can be reused for multiple requests. After completing all of your requests, call
+        // the `InstancesClient.close()` method on the client to safely
+        // clean up any remaining background resources.
```

Comment:
In C#, it wouldn't be IntancesClient.close(), it would be client.Dispose(), but none of these client classes are disposable (probably because they don't need to be) so, this part of the comment is not accurate. Here and everywhere.

```suggestion
        // Initialize client that will be used to send requests. This client only needs to be created
        // once, and can be reused for multiple requests.
```